spec/unit/typesafe/config/conf_parser_spec.rb

```diff
@@ -781,20 +781,26 @@ def assert_comments_at_path_index(comments, conf, path, index)
   # Skipping 'includeURLHeuristically' because we don't support URLs
   # Skipping 'includeURLBasenameHeuristically' because we don't support URLs
 
-  it "acceptBOMStartingFile" do
-    skip("BOM not parsing properly yet; not fixing this now because it most likely only affects windows") do
-      # BOM at start of file should be ignored
-      conf = Hocon::ConfigFactory.parse_file(TestUtils.resource_file("bom.conf"))
-      expect(conf.get_string("foo")).to eq("bar")
-    end
+  it "doesnotacceptUTF8FileNames" do
+    expect { Hocon::ConfigFactory.parse_file(TestUtils.resource_file("ᚠᛇᚻ.conf")) }.to raise_error
   end
```

Comment:
Is the intent of this test to capture something that we should fix in the future?  If so, should we use an assertion that we'd expect to pass, and have a `skip` here?